include-build/roborazzi-core/src/commonJvmMain/kotlin/com/github/takahirom/roborazzi/CaptureResults.kt

```diff
@@ -52,22 +52,22 @@ data class CaptureResults(
           when (image) {
             is CaptureResult.Added -> {
               append("<td class=\"$fileNameClass\" style=\"$fileNameStyle\">${image.actualFile.name}</td>")
-              append("<td class=\"$imgClass\"><img $imgAttributes src=\"${image.actualFile.pathFrom(reportDirectoryPath)}\"/></td>")
+              append("<td class=\"$imgClass\"><img $imgAttributes class=\"modal-trigger\" src=\"${image.actualFile.pathFrom(reportDirectoryPath)}\" data-alt=\"${image.goldenFile.name}\"/></td>")
```

Comment:
How about adding class=\"modal-trigger\" to imgAttributes for consistency?

Reply:
Yup, that makes a lot of sense [ce2037b](https://github.com/takahirom/roborazzi/pull/169/commits/ce2037bbcc8b88e541b99db80aab601cfab839b0)